### PR TITLE
Unit test documentation

### LINKED ISSUE #9: Documentation of unit tests
We should create and add the documentation of the unit tests.

### PR DESCRIPTION
**Pull request by Cuslika**
fixed #9

### CODE REVIEW

**Comment by Cuslika** [comment]
Fixes #9

File: doc/unittest.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Unit tests
+
+We tested those files, which have the most impact on the site. We used jest, which is the most popular javascript testing tool.
```
Comment (jolomarci) [inline doc/unittest.md:3]: > We tested those files

I think here 'the' would be better than 'those'

File: doc/unittest.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+We tested the following: 
+
+- error sending: sendError is used to send different hhtp errors in the backend
```
Comment (jolomarci) [inline doc/unittest.md:7]: > hhtp errors

typo :/

**Review by jolomarci — CHANGES_REQUESTED** [review]

**Review by jolomarci — APPROVED** [review]
looks good

**Pull request merged**